### PR TITLE
Update code generating help for species statistics

### PR DESCRIPTION
**Pull request by azangru**
## Description
Show the species name agains compara coverage statistic has been calculated, in the tooltip on the Species page.

## Related JIRA Issue(s)
https://www.ebi.ac.uk/panda/jira/browse/ENSWBSITES-2288

## Deployment URL(s)
http://compara-stats-help.review.ensembl.org

### CODE REVIEW

File: src/content/app/species/state/general/speciesGeneralHelper.ts [hunk 1]
```diff
@@ -199,6 +197,15 @@ type StatsFormattingOption = {
...
   helpText?: string;
+  getHelpText?: (allStats: SpeciesStatistics) => string;
+};
+
+const getComparaCoverageHelpText = (allStats: SpeciesStatistics) => {
+  const referenceSpeciesName = allStats.homology_stats.reference_species_name;
+  if (!referenceSpeciesName) {
+    return '';
+  }
+  return `Proportion of genes with orthologs with ${referenceSpeciesName}`;
```
Comment (imransl) [inline src/content/app/species/state/general/speciesGeneralHelper.ts:208]: Shouldn't it be `orthologues` instead of `orthologs` here?

Comment (azangru) [reply src/content/app/species/state/general/speciesGeneralHelper.ts:208]: This is the text from the ticket (see [ENSWBSITES-2288](https://www.ebi.ac.uk/panda/jira/browse/ENSWBSITES-2288)). I don't think we do -ues for orthologs, homologs, paralogs, etc.

Comment (imransl) [reply src/content/app/species/state/general/speciesGeneralHelper.ts:208]: I only see `orthologues' in the [current site](https://plants.ensembl.org/Arabidopsis_thaliana/Gene/Compara_Ortholog/pan_compara?g=AT5G06870;r=5:2133918-2135168;t=AT5G06870.1#list_no_ortho) as well as the text used for the same tooltip in the [beta site](https://beta.ensembl.org/species/grch38)

Comment (imransl) [reply src/content/app/species/state/general/speciesGeneralHelper.ts:208]: As discussed during the scrum today, let's leave it as it is for now.

**Review by imransl — APPROVED** [review]

**Pull request merged**